### PR TITLE
WW-160 send email only for en authors

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/WW-160

### CODE REVIEW

File: extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php [hunk 1]
```diff
@@ -169,18 +169,23 @@ private function sendMail( $revisionId ) {
...
-				'revisionUrl' => $editedPageTitle->getFullURL( [
-					'diff' => $revision->getId()
-				] )
-			];
-
-			$this->app->sendRequest( 'Email\Controller\ContributionAppreciationMessageController', 'handle', $params );
+			$diffAuthor = \User::newFromId( $revision->getUser() );
+
+			// we want to send appreciation email for en users only
+			if ( $diffAuthor && $diffAuthor->getGlobalPreference( 'language' ) == 'en' ) {
```
Comment (sebastianmarzjan) [inline extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php]: Please extract the language-specific part of condition from here and https://github.com/Wikia/app/pull/11083/files#diff-dd7892d407f19b1887cafc528350ea56L110 to a single place, i.e.:

```
isSuportedAppreciationLang($lang) 
```

**Comment by sebastianmarzjan** [comment]
LGTM
Is there a ticket for the failing test?

**Comment by rybmat** [comment]
LGTM

**Comment by dianafa** [comment]
@SebastianMarzjan Ads are investigating this

**Pull request merged**